src/pages/index.js

```diff
@@ -29,19 +26,8 @@ const GettingStartedLink = g(ExternalLink)({
 
 const IndexPage = ({ theme }) => (
   <div>
-    <RibbonContainer
-      color="white"
-      textAlign="center"
-      backgroundColor="#F9F9F9"
-      padding={`${rhythm(2)} ${rhythm(3 / 4)}`}
-    >
-      <g.Img
```

Comment:
I kind of liked the logo. Did we remove this for accessibility? If so we can do both. If you think it looks better, I'm ok with that.

Reply:
Just felt like a lot having twice so close together